AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_batch_tests.py

```diff
@@ -694,9 +690,14 @@ def test_validateNestedPreloadDependency_Found(self, asset_processor, ap_setup_f
 
         success, output = asset_processor.batch_process(capture_output=True, expect_failure=False)
         log = APOutputParser(output)
-        for _ in log.get_lines(-1, ["Preload circular dependency detected", "testa.prefab"]):
+        for _ in log.get_lines(-1, ["Preload circular dependency detected", "testa.auto_test_asset"]):
             error_line_found = True
 
+        if not error_line_found:
+            for line in output:
+                print(line)
+
+
```

Comment:
is this debug code?
since this is done if "not error line found"

Reply:
Same as above